plugins/awscdk/plugin.go

```diff
@@ -0,0 +1,22 @@
+package awscdk
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/schema"
+)
+
+func New() schema.Plugin {
+	return schema.Plugin{
+		Name: "awscdk",
+		Platform: schema.PlatformInfo{
+			Name:     "AWS CDK",
+			Homepage: sdk.URL("https://aws.amazon.com/cdk/"),
+		},
+		Credentials: []schema.CredentialType{
+			AccessKey(),
```

Comment:
I'd expect this to have `aws.AccessKey()` or something along these lines so that it completely re-uses the credential, rather than specify its own version of the credential, since both AWS CLI and AWS CDK CLI are using the exact same AWS SDK to load the credential. It's the same credential definition in AWS, so ideally it could also have a single credential definition in the plugin ecosystem.

Reply:
I simplified the code so that the `cdk` executable is defined within the existing `aws` directory, rather than within a new plugin directory, which indirectly addresses your comment I believe :)